src/libraries/System.Text.Json/tests/System.Text.Json.Tests/System.Text.Json.Tests.csproj

```diff
@@ -264,6 +264,9 @@
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
     <Compile Include="$(CommonPath)System\Buffers\ArrayBufferWriter.cs" Link="CommonTest\System\Buffers\ArrayBufferWriter.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="TrimmingTests\IsReflectionEnabledByDefaultFalse.cs" />
```

Comment:
It feels odd that we are putting this one in the unit tests, but not the rest of the TrimmingTests. Do we need this?

Reply:
Good catch, this seems to have survived from the earlier structure I was using.